src/js/cookieManager.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Check if cookie by provided key is exists. Return true, if yes, false, if no.
+ * @param  {} key - Key of cookie.
+ */
+export function CookieExists(key) {
+    if (document.cookie.split(';').filter((item) => item.trim().startsWith(`${key}=`)).length)
+        return true;
+    else
+        return false;
+}
+/**
+ * Set cookie on path "/"
+ * @param  {} key - Key of cookie.
+ * @param  {} value - Value of cookie.
+ */
+export function SetCookie(key, value) {
+    document.cookie=`${key}=${value}; path=/`;
+}
+/**
+ * Get token cookie record by default path
+ */
+export function GetTokenCookieValue() {
```

Comment:
Можно назвать просто
```suggestion
export function getToken() {
```

Reply:
Готово